src/Feature/PageContent/code/GameStoreProject.Feature.PageContent/serialization/GameStoreProject.Feature.PageContent/content/Dictionary.yml

```diff
@@ -9,6 +9,9 @@ SharedFields:
   Value: |
     {0A2847E6-9885-450B-B61E-F9E6528480EF}
     {A87A00B1-E6DB-45AB-8B54-636FEC3B5523}
```

Comment:
I see that serialized Content data is located under the Feature project could you please move it to the Project